core/shared/src/main/scala/scalaz/zio/Promise.scala

```diff
@@ -176,24 +168,20 @@ class Promise[E, A] private (private val state: AtomicReference[State[E, A]]) ex
 }
 object Promise {
 
-  /**
-   * Makes a new promise.
+  /** Makes a new promise.
```

Comment:
I would change your IntelliJ IDEA configuration to insert a newline after `/**`, then you can just have it format the Scaladoc; you can also make IntelliJ IDEA use the `.scalafmt` so it knows the "right" way to format code.